package-lock.json

```diff
@@ -344,6 +424,11 @@
         "mermaid": "^8.13.8"
       }
     },
+    "abbrev": {
```

Comment:
It seems that this small package abbrev was published 5 years ago https://www.npmjs.com/package/abbrev . Does it still work? Or does it need this https://github.com/npm/abbrev-js/pull/34 ? What is the chance of getting new version of 5 year old package?

Reply:
Not sure. It's there as a dependency for `@mapbox/node-pre-gyp` package. The latter was introduced by @xet7 in the commit `4e72dadc6` made on Sat Oct 9 17:00:37 2021 +0300.